### PR TITLE
Add legal change request

### PR DESCRIPTION
**Pull request by avsavsavs**
Changed the last sentence of the question "Is cowswap secure to use?" in line 310 as requested per the legal team. Changed the wording and added a link to our own website terms and conditions (hopefully i put the link correctly).

# Summary

No previous issue

New sentence + link to terms and conditions

  # Background

Change Requested by the legal team

### CODE REVIEW

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -322,7 +322,14 @@ export default function Faq() {
               audited
             </ExternalLink>
             . Whilst CowSwap has taken a major step forward in terms of security and stability, as with other crypto
-            protocols or dapps, it&#39;s recommended to use it at <strong>your own risk</strong>.
+            protocols or dapps, your use is at your own risk. <strong> Please review our terms and conditions </strong>
+            <ExternalLink
+              href="https://cowswap.exchange/#/terms-and-conditions"
```
Comment (alfetopito) [inline src/custom/pages/Faq/index.tsx:327]: What about making this not open in a new page?

Comment (avsavsavs) [reply src/custom/pages/Faq/index.tsx:327]: Sounds good @alfetopito, used the external link right above as a reference for the code.

**Review by anxolin — APPROVED** [review]
Other than Leandro's comment. Approved

**Comment by elena-zh** [comment]
It would be great to add a period in the end of the sentence
image

Also, can we add a link to 'Terms and conditions' phrase itself instead of 'audited' word?

**Comment by anxolin** [comment]
Addressing the issues:

Also the label of the link was wrong. Merging this and cherrypicking into a hotfix. Let me know if something is not right...

**Comment by elena-zh** [comment]
Changes LGTM. Only small issue is left: it would be nice to make 'Terms and conditions' link in bold in order to match the whole sentence style. Reported in  #1450

**Pull request merged**